slack/Module.md

```diff
@@ -0,0 +1,55 @@
+## Overview
+This module allows you to access the Slack Web API through Ballerina. This module provides the capability
+to query information and perform some actions in a Slack workspace.
+
```

Comment:
The module version is not mentioned.

Reply:
Yeah. For all the connectors, we specify the version of the API.
But for Slack, they haven't any version for their apis.
Thats why we couldn't specify the API version